newm-shared/src/main/kotlin/io/newm/shared/ktx/StringExt.kt

```diff
@@ -37,6 +37,8 @@ private object DummyURLHandler : URLStreamHandler() {
     }
 }
 
+fun String.orNull(): String? = takeIf { isNotEmpty() }
```

Comment:
Change to `isNotBlank()`. If they just have nothing but a space in there, we should also reset the field to `null`.

Reply:
Good call! Done.